scripts/skill_auditor/metrics_extractor.py

```
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Ruff (0.14.5)

Shebang is present but file is not executable

(EXE001)